src/coreComponents/physicsSolvers/contact/SolidMechanicsEmbeddedFractures.cpp

```diff
@@ -537,83 +537,90 @@ real64 SolidMechanicsEmbeddedFractures::calculateResidualNorm( real64 const & ti
 
   if( !m_useStaticCondensation )
   {
+    real64 const fractureResidualNorm = calculateFractureResidualNorm( domain, dofManager, localRhs );
```

Comment:
didn't really check anything here, just introduced `calculateFractureResidualNorm` function

Reply:
so same code just in a different function, right?

Reply:
yes, just moved into a function, similar to conforming code